ffi/bindings/dotnet/examples/server/Program.cs

```diff
@@ -142,7 +143,7 @@ private static void RunServer(Server server)
                         // ANCHOR_END: update_coil
                         break;
                     case "udi":
-                        server.Update(1, new DatabaseCallback((db) =>
+                        server.UpdateDatabase(1, DatabaseCallback.create((db) =>
```

Comment:
This can be transformed into an Action<>, check out the DNP3 outstation in C#.

Reply:
Oh indeed, I've updated the example.